testdata/jsight_0.3/ser-to-openapi/req.japi.http_method.query_0.3/query_user_type.openapi.todo

```diff
@@ -0,0 +1,199 @@
+{
+  "openapi": "3.0.3",
+  "info": {
+    "title": "",
+    "version": ""
+  },
+  "components": {
+    "schemas": {
+      "queryUserTypeBasic": {
+        "type": "object",
+        "properties": {
+          "first": {
+            "type": "integer",
+            "example": 123
+          },
+          "second": {
+            "type": "string",
+            "example": "value"
+          }
+        },
+        "required": [
+          "first",
+          "second"
+        ],
+        "additionalProperties": false
+      },
+      "queryUserTypeWithAllTypesOfParameters": {
+        "type": "object",
+        "properties": {
+          "prim1": {
+            "type": "integer",
+            "example": 1
+          },
+          "prim2": {
+            "type": "string",
+            "example": "value"
+          },
+          "arr": {
+            "type": "array",
+            "items": {
+              "type": "integer"
+            }
```

Comment:
The array is incorrectly converted. `anyOf` must be used, see https://j-sight.atlassian.net/wiki/spaces/JAPI/pages/402915358#array.

Reply:
Fixed.